Solutions/CyberArkEPM/Parsers/CyberArkEPM

```diff
@@ -0,0 +1,83 @@
+// Usage Instructions: 
```

Comment:
@averbn , Documentation links should not include locale: Solutions/CyberArkEPM/Parsers/CyberArkEPM, please check and update.
Add **.txt** extenstion to the parser file, thanks.

Reply:
Done